controllers/cart.ts

```diff
@@ -1,11 +1,14 @@
 import * as zod from "zod";
+// PRO: Ramda is a really cool library, but could probably
```

Comment:
The standard reduce function would definitely cover the scenarios we have. One thing that I like about the ramda version though is that we can create a helper reusable function (L25) that is smaller in size and can potentially be reused in ramda `pipe` or `compose`.